package/common_init/overlay/etc/init.d/common.sh

```diff
@@ -104,6 +104,37 @@ configure_file_resource2()
   chmod "$perm" "$path"
 }
 
+configure_logrotate_file()
+{
+  local conf_tag=$1 ; shift # must be unique!
+  local rotate_file=$1 ; shift
+
+  _validate_not_empty conf_tag     || return 1
+  _validate_not_empty rotate_file  || return 1
+
+  path="/etc/logrotate.d/$conf_tag"
+  [[ ! -e "$path" ]] || return 1
+
+  touch "$path"
+  echo "$rotate_file {"     >  "$path"
+  echo "      su root root" >> "$path"
+  echo "      size 5k"      >> "$path"
+  echo "      copytruncate" >> "$path"
+  echo "      rotate 5"     >> "$path"
+  echo "      missingok"    >> "$path"
```

Comment:
I think you are right, still investigating. Will report back but right now this is likely wrong

Reply:
So since we are simply piping to file for the template processes, it looks like `copytruncate` works as expected. The contents of the file are simply copied a new file with the rollover title, and the output continues to pipe to same file, after the contents are truncated to zero.

Reply:
👍 cool